db_migrations/01-pending.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE pendingpatches_tmp(id INTEGER PRIMARY KEY, pdate TEXT,
```

Comment:
Can we have this named `pendingpatches_new` for clarity?

Not necessary for my ACK.